ETL_week_2.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+import sqlalchemy as sql
+
+# 1. Extract
+
+db = sql.create_engine("postgresql://postgres@localhost/alex")
+
+# Create dataframes from data tables
+employee_id = pd.read_sql_table("employee_id", db)
+office_info = pd.read_sql_table("office_info", db)
+product_inventory = pd.read_sql_table("product_inventory", db)
+production = pd.read_sql_table("production", db)
+order_log = pd.read_sql_table("order_log", db)
+customers = pd.read_sql_table("customers", db)
+
+# 2. Transform
+
+# Create fact table
+df_1 = pd.merge(employee_id['employee_number','office_code'], office_info['office_code','city'], on ='office_code')
```

Comment:
Having variables with numbers at the end should be an indication that there might be a nicer way to do this! You can either replace variable names or just chain methods!